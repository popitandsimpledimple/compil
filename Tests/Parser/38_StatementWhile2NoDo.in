var a: integer = 0;
begin
while a < 10 
    begin
    a := 1 + a;
    writeln(a);
    end;
end.