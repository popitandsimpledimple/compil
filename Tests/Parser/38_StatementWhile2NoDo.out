(4,5) Expected 'DO' Found 'BEGIN'